screens/OrdersSummaryRow.js

```diff
@@ -455,6 +458,7 @@ class OrdersSummaryRow extends React.Component {
           <RealTimeOrderUpdate
             topics={`/topic/order/${order?.orderId}`}
             handleOnMessage={this.handleOnMessage}
+            handleOnConnect={this.handleOnConnect}
```

Comment:
I was thinking that RealtimeOrderUpdate can provide default handleOnConnect function so it doesn't have to be provided on every usage.